src/mapml/handlers/ContextMenu.js

```diff
@@ -723,6 +770,85 @@ export var ContextMenu = L.Handler.extend({
         L.DomEvent.stop(e);
         this._focusOnLayerControl();
       }
+    } else if (key === 38) { //up arrow
+      if (!this._coordMenu.hasAttribute('hidden')&&document.activeElement.shadowRoot.activeElement.innerHTML === 'Map') {
```

Comment:
This will cause issues when the contextmenu is in a different language, through the extension.

There is probably a better way to this, take a look at how `_layerMenuTabs` works.